arviz/plots/distplot.py

```diff
@@ -88,14 +88,14 @@ def plot_dist(
         Ignored for 2D KDE
     rug_kwargs : dict
         Keywords passed to the rug plot. Ignored if rug=False or for 2D KDE
-        Use `space` keyword (float) to control the position of the rugplot. The larger this number
+        Use ``space`` keyword (float) to control the position of the rugplot. The larger this number
         the lower the rugplot.
```

Comment:
All the kwarg type arguments should have references to the functions they are eventually passed to. I'll do a couple of examples with links here, but all need to be updated with the same pattern. 

`rug_kwargs` is passed to `plot_dist` (defined in https://github.com/arviz-devs/arviz/blob/main/arviz/plots/distplot.py#L28) and then on to the backend plotting function (see [this line](https://github.com/arviz-devs/arviz/blob/main/arviz/plots/distplot.py#L195) and [these lines](https://github.com/arviz-devs/arviz/blob/main/arviz/plots/distplot.py#L211-L212)). 

### matplotlib
The matplotlib version passes it to `plot_kde` (https://github.com/arviz-devs/arviz/blob/main/arviz/plots/distplot.py#L211-L212). You'll see that in some cases, the kwargs are modified with setdefaults or things like that. You can ignore all that except for **pop** clauses, the only part relevant for these PRs is where they are passed to, no need to understand the code either. It is enough to say there are pop clauses and ask for help if you see them, again, no need to understand exactly what the code is doing.

### bokeh
Bokeh version also passes it to `plot_kde`. 

---

Therefore, in this case, the only change needed is adding a sentence to the argument description along the lines of 

    Passed to :func:`arviz.plot_kde` as ``rug_kwargs``

and it would be `plot_kde` that would have more detail on that.